laws/js/src/main/scala/algebra/laws/Platform.scala

```diff
@@ -7,9 +7,10 @@ import Prop.{False, Proof, Result}
 
 private[laws] object Platform {
 
-  // Scala-js does not implement the Serializable interface, so we just retuen true.
+  // Scala-js does not implement the Serializable interface, so we just return true.
   @inline
-  def serializable[M](m: M): (String, Prop) = "serializable" -> Prop { _ =>
-    Result(status = Proof)
-  }
+  def serializable[M: IsSerializable](m: M): (String, Prop) =
```

Comment:
This seems a little bit confusing to me, because it looks like `M` must be serializable in order to call this, when in fact the `IsSerializable` instance could be `IsNot`. I'm at a loss for names that seem any better though.